src/Symfony/Component/Translation/Formatter/PluralMessageFormatterInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\Formatter;
+
+/**
+ * @author Abdellatif Ait boudad <a.aitboudad@gmail.com>
+ */
+interface PluralMessageFormatterInterface extends MessageFormatterInterface
```

Comment:
Java API defines a `ChoiceFormat` message formatter ([see details](https://docs.oracle.com/javase/7/docs/api/java/text/ChoiceFormat.html#ChoiceFormat(java.lang.String))) Maybe it's a better alternative to `PluralMessageFormatter` ?